src/test/regress/expected/isolation_ensure_dependency_activate_node.out

```diff
@@ -8,7 +8,7 @@ step s1-print-distributed-objects:
     SELECT nodename, nodeport, isactive FROM master_add_node('localhost', 57638);
 
     -- print an overview of all distributed objects
-    SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object;
+    SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object GROUP BY 1;
```

Comment:
What's the benefit of the GROUP BY?

Reply:
should have been an `ORDER BY`, thanks.

The test failed once due to different ordering.